main.js

```diff
@@ -0,0 +1,11 @@
+const tree = document.getElementById('tree');
+
+function hideElementSibling(event) {
+    const {target: tagName} = event;

```

Comment:
Why do you rename it? It is not a tagName?
also add spaces same as in object literals 
```javascript
    const { target: tagName } = event;
```